src/foam/dao/SequenceNumberDAO.js

```diff
@@ -128,10 +140,7 @@ foam.CLASS({
         synchronized (this) {
           long id = (long) getProperty_().get(obj);
           if ( id == 0 ) {
-            getProperty_().set(obj, getValue_());
-            setValue_(getValue_() + 1);
-          } else if ( id >= getValue_() ) {
-            setValue_(id + 1);
+            getProperty_().set(obj, Long.parseLong(getUIDGenerator().generate(), 16));
```

Comment:
the old `getValue_()` also considers the startingValue property, now they're not being used.

UID generator will return 12 to 30 chars hex string, would it reduce the number of seqNum we can have and might also overflow Java and Javascript long?